packages/bigtable/src/index.js

```diff
@@ -262,267 +289,262 @@ function Bigtable(options) {
-    'projects/{projectId}/zones/{zone}/clusters/{cluster}',
-    options
-  );
+  var adminBaseUrl = 'bigtableadmin.googleapis.com';
 
   var config = {
     baseUrl: 'bigtable.googleapis.com',
     service: 'bigtable',
-    apiVersion: 'v1',
+    apiVersion: 'v2',
     protoServices: {
-      BigtableService: googleProtoFiles.bigtable.v1,
-      BigtableTableService: {
-        path: googleProtoFiles.bigtable.admin,
-        service: 'bigtable.admin.table'
+      Bigtable: googleProtoFiles.bigtable.v2,
+      BigtableTableAdmin: {
+        baseUrl: adminBaseUrl,
+        path: googleProtoFiles.bigtable.admin.v2.table,
+        service: 'bigtable.admin'
+      },
+      BigtableInstanceAdmin: {
+        baseUrl: adminBaseUrl,
+        path: googleProtoFiles.bigtable.admin.v2.instance,
+        service: 'bigtable.admin'
+      },
+      Operations: {
+        baseUrl: adminBaseUrl,
+        path: googleProtoFiles('longrunning/operations.proto'),
+        service: 'longrunning',
+        apiVersion: 'v1'
       }
     },
     scopes: [
       'https://www.googleapis.com/auth/bigtable.admin',
-      'https://www.googleapis.com/auth/bigtable.data'
+      'https://www.googleapis.com/auth/bigtable.data',
+      'https://www.googleapis.com/auth/cloud-platform'
     ],
     userAgent: PKG.name + '/' + PKG.version
   };
 
   common.GrpcService.call(this, config, options);
+
+  this.projectName = 'projects/' + this.projectId;
 }
 
 util.inherits(Bigtable, common.GrpcService);
 
 /**
- * Formats the full table name into a user friendly version.
+ * Create a Compute instance.
  *
- * @private
- *
- * @param {string} name - The formatted Table name.
- * @return {string}
+ * @resource [Creating a Compute Instance]{@link https://cloud.google.com/bigtable/docs/creating-compute-instance}
  *
- * @example
- * Bigtable.formatTableName_('projects/p/zones/z/clusters/c/tables/my-table');
- * // => 'my-table'
- */
-Bigtable.formatTableName_ = function(name) {
-  if (name.indexOf('/') === -1) {
-    return name;
-  }
-
-  var parts = name.split('/');
-  return parts[parts.length - 1];
-};
-
-/**
- * Create a table on your Bigtable cluster.
- *
- * @resource [Designing Your Schema]{@link https://cloud.google.com/bigtable/docs/schema-design}
- * @resource [Splitting Keys]{@link https://cloud.google.com/bigtable/docs/managing-tables#splits}
- *
- * @throws {error} If a name is not provided.
- *
- * @param {string} name - The name of the table.
- * @param {object=} options - Table creation options.
- * @param {object|string[]} options.families - Column families to be created
- *     within the table.
- * @param {string} options.operation - Operation used for table that has already
- *    been queued to be created.
- * @param {string[]} options.splits - Initial
- *    [split keys](https://cloud.google.com/bigtable/docs/managing-tables#splits).
+ * @param {string} name - The unique name of the instance.
+ * @param {object} options - Instance creation options.
```

Comment:
If there are required arguments, this should be `config` instead of `options` (i.e., they're not *option*al has been my distinction thus far in the API). We should also throw if the arguments that are required aren't provided.